modules/tinymce/src/core/test/ts/browser/bookmark/BookmarksTest.ts

```diff
@@ -223,4 +223,53 @@ describe('browser.tinymce.core.bookmark.BookmarksTest', () => {
     assertApproxRawContent(editor, '<p>abc</p>');
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
   }));
+
+  context('Get bookmark should work if the first element of the content is a comment', () => {
+    const outsideButton: SugarElement<HTMLButtonElement> = SugarElement.fromHtml('<button id="getBookmarkButton">Get Bookmark</button>');
+    const setupElement = () => {
+      const element = SugarElement.fromTag('textarea');
+      Insert.append(SugarBody.body(), outsideButton);
+
+      return {
+        element,
+        teardown: () => Remove.remove(outsideButton)
+      };
+    };
```

Comment:
This doesn't ever append and remove the textarea to the body so that's not going to initialize properly.

Reply:
I maybe missing something about this API, if I understood correctly `setupElement` is passed down to [setupHooks](https://github.com/tinymce/tinymce/blob/develop/modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinyHooks.ts#L94), than the element in the return obj is passed as `elementOpt` to [Loader.setup](https://github.com/tinymce/tinymce/blob/develop/modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinyHooks.ts#L54) that use it as [target](https://github.com/tinymce/tinymce/blob/develop/modules/mcagar/src/main/ts/ephox/mcagar/loader/Loader.ts#L53).
And it is `Loader.setup` that [append](https://github.com/tinymce/tinymce/blob/develop/modules/mcagar/src/main/ts/ephox/mcagar/loader/Loader.ts#L58) and [remove](https://github.com/tinymce/tinymce/blob/develop/modules/mcagar/src/main/ts/ephox/mcagar/loader/Loader.ts#L63) the element.

I tried to remove the fix and the test broke as expected, so it seems that the current behavior is not a false positive, do I miss something? ^^"

Reply:
done